src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/CopyAnalysis/CopyAnalysis.CoreCopyAnalysisDataDomain.cs

```diff
@@ -22,21 +22,21 @@ public CoreCopyAnalysisDataDomain(AbstractValueDomain<CopyAbstractValue> valueDo
                 _getDefaultCopyValue = getDefaultCopyValue;
             }
 
-            public override CoreCopyAnalysisData Merge(CoreCopyAnalysisData map1, CoreCopyAnalysisData map2)
+            public override CoreCopyAnalysisData Merge(CoreCopyAnalysisData value1, CoreCopyAnalysisData value2)
```

Comment:
I'd rather just revert changes to this method and add a source suppression here.